packages/esm-patient-test-results-app/src/panel-timeline/helpers.tsx

```diff
@@ -116,23 +116,40 @@ export const TimelineCell: React.FC<{
   );
 };
 
-export const RowStartCell = ({ title, range, units, shadow = false, testUuid }) => (
-  <div
-    className={styles['row-start-cell']}
-    style={{
-      boxShadow: shadow ? '8px 0 20px 0 rgba(0,0,0,0.15)' : undefined,
-    }}
-  >
-    <span className={styles['trendline-link']}>
-      <ConfigurableLink to={`${testResultsBasePath(`/patient/${patientUuid}/chart`)}/trendline/${testUuid}`}>
-        {title}
-      </ConfigurableLink>
-    </span>
-    <span className={styles['range-units']}>
-      {range} {units}
-    </span>
-  </div>
-);
+export const RowStartCell = ({ title, range, units, shadow = false, testUuid, isString = false }) => {
+  if (isString) {
+    return (
+      <div
+        className={styles['row-start-cell']}
+        style={{
+          boxShadow: shadow ? '8px 0 20px 0 rgba(0,0,0,0.15)' : undefined,
+        }}
+      >
+        <span className={styles['trendline-link']}>{title}</span>
+        <span className={styles['range-units']}>
+          {range} {units}
+        </span>
+      </div>
+    );
+  }
+  return (
+    <div
+      className={styles['row-start-cell']}
+      style={{
+        boxShadow: shadow ? '8px 0 20px 0 rgba(0,0,0,0.15)' : undefined,
+      }}
+    >
+      <span className={styles['trendline-link']}>
```

Comment:
This is very similar to beforehand - its either the `title` or the `ConfigurableLink` with `title`. Maybe we can save the whole code duplication above?